integration/hive/hive-openlineage-hook/src/main/java/io/openlineage/hive/hooks/HiveOpenLineageHook.java

```diff
@@ -95,17 +90,14 @@ public void run(HookContext hookContext) throws Exception {
       OpenLineageContext olContext =
           OpenLineageContext.builder()
               .openLineage(new OpenLineage(Versions.OPEN_LINEAGE_PRODUCER_URI))
-              .queryString(hookContext.getQueryPlan().getQueryString())
-              .semanticAnalyzer(semanticAnalyzer)
-              .eventTime(Instant.ofEpochMilli(System.currentTimeMillis()).atZone(ZoneId.of("UTC")))
+              .openLineageConfig(
+                  HiveOpenLineageConfigParser.extractFromHadoopConf(hookContext.getConf()))
+              .hookContext(hookContext)
+              .hadoopConf(hookContext.getConf())
```

Comment:
since we remove the fields that are in `hookContext` we could remove the `hadoopConf` as well?